packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -501,3 +506,15 @@ export enum CheckboxVisibility {
    */
   hidden = 2
 }
+
+export type IDetailsListStyleProps = Required<Pick<IDetailsListProps, 'theme'>> &
+  Pick<IDetailsListProps, 'className'> & {
+    isHorizontalConstrained?: boolean;
+    compact?: boolean;
+    isFixed?: boolean;
+  };
+
+export interface IDetailsListStyles {
+  root: IStyle;
+  focusZone: IStyle;
```

Comment:
And arguably these too (again there are various examples of components which do complete documentation). But minimally the styleprops.

(Look at OverflowSet.types.ts for example...)

Reply:
I'm documenting the styleprops for now - DL subcomponents are not expected to be used directly by others. They're actually kind of a distraction. There are only two styles here and they seem to be pretty self-explanatory.